dev_guide/deployments.adoc

```diff
@@ -376,17 +376,19 @@ strategy:
   type: Recreate
   recreateParams: <1>
     pre: {} <2>
+    mid: {}
     post: {}
 ----
 
 <1> `*recreateParams*` are optional.
-<2> `*pre*` and `*post*` are both link:#lifecycle-hooks[lifecycle hooks].
+<2> `*pre*`, `mid`, and `*post*` are both link:#lifecycle-hooks[lifecycle hooks].
```

Comment:
For consistency w/ the other elements in the list, the markup should be: BACKTICK ASTERISK mid ASTERISK BACKTICK.